pkg/ccl/streamingccl/logical/logical_replication_writer_processor.go

```diff
@@ -744,7 +749,7 @@ func (lrw *logicalReplicationWriterProcessor) flushBatch(
 		}
 		if !row.IsDeleted() {
 			datums := make([]interface{}, 0, len(row.EncDatums()))
-			err := row.ForEachColumn().Datum(func(d tree.Datum, col cdcevent.ResultColumn) error {
+			err := row.ForAllColumns().Datum(func(d tree.Datum, col cdcevent.ResultColumn) error {
```

Comment:
what's the motivation here? does foreach only return one column per col family? the docs were slim.

Reply:
I _think_ ForEachColumn only returns columns from the column family whereas we need to also see columns that were part of the key since we need them for the insert statement.

Reply:
i see, thanks!